docs/product/crons/monitor-details.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Monitor Details
+sidebar_order: 1
+description: "Learn how to navigate the Monitor Details page to help you understand your recurring job's overall health."
+---
+
+The **Monitor Details** page is where you'll find a daily historical bar chart showing successful, failed, and missed check-ins and a line chart of the runtime average for the job execution.
```

Comment:
The timeline is here too right?

Also it's actually an Area chat for the durations